sonar-helm-for-iac/main.go

```diff
@@ -91,7 +104,7 @@ func newHandleID(name string, content string) (int, error) {
 func executeWithValues(templateId int, valuesFileContent string) (string, error) {
 	valuesMap, err := yamlToMap(valuesFileContent)
 	if err != nil {
-		return "", err
+		return "", fmt.Errorf("error parsing values file: %w", err)
```

Comment:
```suggestion
		return "", fmt.Errorf("Error during parsing of values file: %w", err)
```

Reply:
The reason to make these messages different from what we're used to in Java, is the way they are later handled. We basically get a long string in a form `message1: message2: ... : messageN`.  And eventually we'll see something like
`DEBUG: Evaluation error in Go library: error parsing values file: error converting YAML to JSON: yaml: line 6: did not find expected key`.
But thinking about it, they can be made more verbose. However, Go plugin suggests to start them with lowercase letter, and IMO it makes sense considering all the information is eventually concatenated

Reply:
I think a bit more verbose would be nice, to better pinpoint the errors (and making it more easily readable).
But I agree, they can get quite long. (But I think that's okay, because it's only debug messages for people that are interested).
Starting lower case is ok